tests/purs/failing/MultipleErrorsInOneRow.out

```diff
@@ -0,0 +1,76 @@
+Error 1 of 2:
```

Comment:
Before the error this output only had one error. Now we have two. I see this as a win. But it could also compound if we have a large nested tree of almost errors:
```
{ a :: A, b :: A, c :: A } <=> { a :: A', b :: A', c :: A' }
```
This might make the errors hard to read. I prefer this, but nothing in this world is objective. But here I also assume we stop outputting errors after, like, 10.

Reply:
I don't *think* we have any such limit, at least in the compiler. (Maybe IDE plugins do that filtering.) Would that alter your opinion about this change?

Reply:
There's one problem here with outputting many more errors, but these errors are also a lot simpler than the original error. From reading these errors we can see that we have two errors in the types, which is valid. But for this to be a strict improvement I would want the source spans to be better.